README.md

```diff
@@ -38,14 +38,33 @@ sudo crontab -e
 ### Notes:
 + Please read the license
 + This project can be placed in any directory
-+ This is not the finish product so there will be bugs!
++ This is not the finished product so there will be bugs!
 
 
 ### Getting Started!
-1. Run webpage.py & blinkt_status_display.py
-2. Visit [pi_local_ip_address]:5000 in any browser to edit your status!
-3. You can use any password for the first time you login (but make sure you remember it! - otherwise clear the blinkt_hash file and it will reset)
+1. Run dashboard.py
+2. Visit localhost:5000 in any browser to edit your status! If they are reported as missing, make sure they are powered up.
```

Comment:
I think the dashboard runs on port 5001 by default

Reply:
Correct, my bad